packages/adaptive_scaffold/README.md

```diff
@@ -39,52 +55,81 @@ SlotLayout handles the adaptivity or the changes between widgets at certain Brea
+      // then an unextended NavigationRail with no labels and just icons then an
+      // extended NavigationRail with both icons and labels.
+      primaryNavigation: SlotLayout(
+        config: <Breakpoint, SlotLayoutConfig>{
+          Breakpoints.medium: SlotLayout.from(
+            inAnimation: AdaptiveScaffold.leftOutIn,
+            key: const Key('pnav1'),
+            builder: (_) => AdaptiveScaffold.standardNavigationRail(
+                leading: const Icon(Icons.menu),
+                destinations: destinations
+                    .map((_) => AdaptiveScaffold.toRailDestination(_))
+                    .toList()),
+          ),
+          Breakpoints.large: SlotLayout.from(
+            key: const Key('pn1'),
+            inAnimation: AdaptiveScaffold.leftOutIn,
+            builder: (_) => AdaptiveScaffold.standardNavigationRail(
+              extended: true,
+              leading: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceAround,
+                children: const <Widget>[
+                  Text('REPLY',
+                      style:
+                          TextStyle(color: Color.fromARGB(255, 255, 201, 197))),
+                  Icon(Icons.menu_open)
+                ],
+              ),
+              destinations: destinations
+                  .map((_) => AdaptiveScaffold.toRailDestination(_))
+                  .toList(),
+              trailing: trailingNavRail,
+            ),
+          ),
+        },
+      ),
+      // Body switches between a ListView and a GridView from small to medium
+      // breakpoints and onwards.
+      body: SlotLayout(
+        config: <Breakpoint, SlotLayoutConfig>{
+          Breakpoints.small: SlotLayout.from(
+            key: const Key('body'),
+            builder: (_) => ListView.builder(
+              itemCount: children.length,
+              itemBuilder: (BuildContext context, int index) => children[index],
+            ),
+          ),
+          Breakpoints.mediumAndUp: SlotLayout.from(
+            key: const Key('body1'),
+            builder: (_) =>
+                GridView.count(crossAxisCount: 2, children: children),
+          )
+        },
+      ),
+      // BottomNavigation is only active in small views defined as under 600 dp
+      // width.
+      bottomNavigation: SlotLayout(
+        config: <Breakpoint, SlotLayoutConfig>{
+          Breakpoints.small: SlotLayout.from(
+            key: const Key('bn'),
+            inAnimation: AdaptiveScaffold.bottomToTop,
+            outAnimation: AdaptiveScaffold.topToBottom,
+            builder: (_) => BottomNavigationBarTheme(
+                data: const BottomNavigationBarThemeData(
+                    selectedItemColor: Colors.black),
+                child: AdaptiveScaffold.standardBottomNavigationBar(
+                    destinations: destinations)),
+          )
+        },
+      ),
+    );
 ```
 ##
 Both of the examples shown here produce the same output:
```

Comment:
Nit: add a period a couple lines under this, after the link: `sharing)` -> `sharing).`